Gems/EMotionFX/Code/Source/Editor/SkeletonModel.cpp

```diff
@@ -194,11 +219,16 @@ namespace EMotionFX
         if (parent.isValid())
         {
             const Node* parentNode = static_cast<const Node*>(parent.internalPointer());
+
+            if (parentNode == m_characterRootNode) {
+                return static_cast<int>(m_skeleton->GetNumRootNodes());
+            }
+
             return static_cast<int>(parentNode->GetNumChildNodes());
         }
         else
         {
-            return static_cast<int>(m_skeleton->GetNumRootNodes());
+            return 1;
```

Comment:
question: Should this maybe be a constant somewhere? 🤔 Not sure just seems like a named value might be easier to understand in context